controller/transfer/cognition/import_wizard.py

```diff
@@ -0,0 +1,628 @@
+        cognition_project_id,
+        "cognition_wizard:prep:RELEVANCE:COMPLETE",
+        organization_id=organization_id,
+    )
+
+    task_list.append(
+        {
+            "project_id": reference_project_id,
+            "task_type": enums.TaskType.TASK_QUEUE_ACTION.value,
+            "action": {
+                "action_type": enums.TaskQueueAction.FINISH_COGNITION_SETUP.value,
+                "cognition_project_id": cognition_project_id,
+            },
+        }
+    )
+
+    __add_start_gates_for(reference_project_id, task_list)
+    __add_start_gates_for(question_project_id, task_list)
+    __add_start_gates_for(relevance_project_id, task_list)
+
+    task_list.append(
+        {
+            "project_id": reference_project_id,
+            "task_type": enums.TaskType.TASK_QUEUE_ACTION.value,
+            "action": {
+                "action_type": enums.TaskQueueAction.FINISH_COGNITION_SETUP.value,
+                "cognition_project_id": cognition_project_id,
+            },
+        }
+    )
+
+    # wait for initial tokenization to finish
+    c = 0
+    while True:
+        time.sleep(1)
+        c += 1
+        if c > 120:
+            ctx_token = general.remove_and_refresh_session(ctx_token, True)
+        if not tokenization_db_bo.is_doc_bin_creation_running(reference_project_id):
+            break
+
+    task_id = task_queue_manager.add_task(
+        reference_project_id, enums.TaskType.TASK_QUEUE, user_id, task_list
+    )
+
+    notification.send_organization_update(
+        cognition_project_id,
+        f"cognition_wizard:task_queue:{task_id}:{len(task_list)}",
+        organization_id=organization_id,
+    )
+    general.remove_and_refresh_session(ctx_token, False)
+
+
+# function called from queue as last entry
+def finish_cognition_setup(
+    cognition_project_id: str,
+) -> None:
+    ctx_token = general.get_ctx_token()
+    cognition_project_item = cognition_project.get(cognition_project_id)
+    if not cognition_project_item:
+        general.remove_and_refresh_session(ctx_token, False)
+        return
+    user_id = str(cognition_project_item.created_by)
+    notification_db_bo.set_notifications_to_not_initial(
+        str(cognition_project_item.refinery_references_project_id), user_id
+    )
+    notification_db_bo.set_notifications_to_not_initial(
+        str(cognition_project_item.refinery_question_project_id), user_id
+    )
+    notification_db_bo.set_notifications_to_not_initial(
+        str(cognition_project_item.refinery_relevance_project_id), user_id
+    )
+
+    cognition_project_item.wizard_running = False
+    organization_id = str(cognition_project_item.organization_id)
+    general.commit()
+    general.remove_and_refresh_session(ctx_token, False)
+    notification.send_organization_update(
+        cognition_project_id,
+        "cognition_prep:state:DONE",
+        organization_id=organization_id,
+    )
+
+
+def __add_websocket_message_queue_item(
```

Comment:
This method ist not used atm, should we keep it for future usage?

Reply:
yes